Storage/SQL/SQLiteBookmarksModel.swift

```diff
@@ -531,6 +531,18 @@ extension SQLiteBookmarks {
             (deleteStructure, args),
         ])
     }
+
+    fileprivate func markBufferBookmarkAsDeleted(_ guid: GUID) -> Success {
+        let insertInPendingDeletions =
+        "INSERT OR IGNORE INTO \(TablePendingBookmarksDeletions) " +
+        "(id, server_modified_at_deletion) " +
+        "SELECT " +
+        "? AS id, " +
```

Comment:
Use SQLite's positional args so you only have to supply the arg once, or project the GUID here.